pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -203,14 +173,6 @@ spec:
         selector:
           - app=kotsadm
         timeout: 10s
-    - copyFromHost:
-        collectorName: kurl-host-preflights
-        name: kots/kurl/host-preflights
-        hostPath: /var/lib/kurl/host-preflights
-        extractArchive: true
-        image: alpine
-        imagePullPolicy: IfNotPresent
-        timeout: 1m
     - configMap:
         collectorName: kurl-current-config
         name: kurl-current-config # NOTE: this will not live under the kots/ directory like other collectors
```

Comment:
Could we find a way to separate out _all_ of the kurl-specific collectors and analyzers into a separate support bundle spec and load/append those only when needed?  For example, these kurl configmap colelctors.  Same would go for the kurl add-ons (weave, rook, etc).

Reply:
I updated the two functions to remove those collectors/analyzers as well.